src/styles/Theme.tsx

```diff
@@ -18,6 +35,11 @@ const theme = createTheme({
     success: {
       main: "#22AA6F",
     },
+    greyButton: palette.augmentColor({
```

Comment:
I wanna see if there's a neater way to do this. I think we should try to reduce the amount of interfaces we have in this file. It also might not be worth making a new class in theme just for 1 grey button

Reply:
For us to include custom colours in Typescript we will need to customize these interfaces for it to recognize any custom colours that we create. 

But yeah honestly it's just one grey button..if we want to use this colour again, or want to revisit augmentation then I think we can start looking into it. 

I resulted in removing the augmentation and passing in the `sx` prop on Button with a custom `styles` that provided that color.

Reply:
We went through this together and ended up getting rid of this color since it's only used in one spot. You're right though, if we end up adding new colors to our palette we'll have to use these interfaces